loleaflet/src/layer/tile/CommentSection.ts

```diff
@@ -71,6 +82,542 @@ class CommentSection {
 
 	}
 
+	private adjustComment (comment: any) {
+		var rectangles, color, viewId;
+		comment.trackchange = false;
+		rectangles = L.PolyUtil.rectanglesToPolygons(L.LOUtil.stringToRectangles(comment.textRange || comment.anchorPos), this.map._docLayer);
+		comment.anchorPos = L.LOUtil.stringToBounds(comment.anchorPos);
+		comment.anchorPix = this.map._docLayer._twipsToPixels(comment.anchorPos.min);
+		viewId = this.map.getViewId(comment.author);
+		color = viewId >= 0 ? L.LOUtil.rgbToHex(this.map.getViewColor(viewId)) : '#43ACE8';
+		if (rectangles.length > 0) {
+			comment.textSelected = L.polygon(rectangles, {
```

Comment:
Is this going to use the old svg overlay layer and not the canvas after all and having a comments section etc is just to sync stuff with other sections specifically the tiles ?